lessons/Early-Math/associative-property/langs/en-us/pages/discover-assoc-cont.adoc

```diff
@@ -0,0 +1,62 @@
+= Discover the Associative Property (2)
+
+++++
+<style>
+  table {grid-template-rows: 3fr 1fr !important;}
+  div.circleevalsexp .value,
+  div.circleevalsexp .studentBlockAnswerFilled { min-width:unset; }
+</style>
+++++
+
+@n Evaluate the two Circles of Evaluation below to help you decided if the Associative Property holds for problems involving *multiplication*:
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="^.^3,^.^3"]
+|===
+|@show{(coe '(* (* 2 5) 10))} | @show{(coe  '(* 2 (* 5 10)))}
+| @ifnotsoln{@math{(2 \times 5) \times 10 = ?}} @ifsoln{@math{(2 \times 5) \times 10 = 100}} | @ifnotsoln{@math{2 \times (5 \times 10) = ?}} @ifsoln{@math{2 \times (5 \times 10) = 100}}
+|===
+
+What do you notice? @fitb{}{@ifsoln{Both Circles of Evaluation lead to 100. The first Circle has the product of 2 and 5 in a nested Circle, while the second}}
+
+@fitb{}{@ifsoln{Circle has the product of 5 and 10 in a nested Circle.}}
+
+@vspace{1ex}
+
+These Circles of Evaluation illustrate the Associative Property of Multiplication. Draw another example of the Associative Property of Multiplication with any three numbers, below. Evaluate your expressions to confirm that they are equivalent.
```

Comment:
Noting that for multiplication you're asking students to draw another example with any 3 numbers. on the previous page you asked them to use 4 numbers. Perhaps stick to 3 on both?